infocmdb/v1/cmdb/cmdb_test.go

```diff
@@ -118,14 +116,16 @@ func (t *testing) MockServer() *httptest.Server {
 }
 
 func (t *testing) getUrl() string {
-	testingURL := os.Getenv("WORKFLOW_TEST_URL")
+	var testingURL string
 	if mocking {
-		ts := t.MockServer()
-		testingURL = ts.URL
-	}
+		mockService := t.MockServer()
+		testingURL = mockService.URL
+	} else {
+		testingURL = os.Getenv("WORKFLOW_TEST_URL")
 
-	if testingURL == "" {
-		log.Fatal(ErrTestingInfocmdbUrlMissing)
+		if testingURL == "" {
+			log.Fatal(ErrTestingInfocmdbUrlMissing)
+		}
```

Comment:
this would be more go-like
```
	testingURL = os.Getenv("WORKFLOW_TEST_URL")
	if mocking {
		ts := t.MockServer()
		testingURL = ts.URL
	}

    if testingURL == "" {
		log.Fatal(ErrTestingInfocmdbUrlMissing)	
    }
```